test/apiv2/20-containers.at

```diff
@@ -207,7 +207,7 @@ t DELETE containers/$cid 204
 t DELETE containers/$cid_top 204
 
 # test the apiv2 create, shouldn't ignore the ENV and WORKDIR from the image
-t POST containers/create '"Image":"'$ENV_WORKDIR_IMG'","Env":["testKey1"]' 201 \
+t POST containers/create '"Image":"'$ENV_WORKDIR_IMG'","Env":["testKey1=foo"]' 201 \
```

Comment:
https://github.com/containers/podman/blob/93af493ccb13b3d93717bf9c805f820c9e3a2fba/test/apiv2/test-apiv2#L154-L163
the line `sed -e 's/\(.*\)=\(.*\)/"\1":"\2"/' <<<$i` will convert `=` to `:`, so don't need `testKey1=foo`